rust/definitions/src/keyring.rs

```diff
@@ -30,6 +30,7 @@
 //!
 use parity_scale_codec::{Decode, Encode};
 use sled::IVec;
+use sp_core::H256;
```

Comment:
Let's replace VerifierKey inner content with H256 as well. Because VerifierKey is just the genesis hash.